ext/Recorder.php

```diff
@@ -16,8 +16,6 @@
  * Saves a screenshot of each step in acceptance tests and shows them as a slideshow on one HTML page.
  * Activated only for suites with WebDriver module enabled.
  *
- *  ![recorder](http://codeception.com/images/recorder.gif)
```

Comment:
No, it's really annoying! Even worse than most ads ;-) And it doesn't add any value. Everybody knows what a screenshot looks like...